hal/src/electron/modem/mdm_hal.cpp

```diff
@@ -118,6 +118,34 @@ inline void CLR_GPRS_TIMEOUT() {
     DEBUG("GPRS WD Cleared, was %d", gprs_timeout_duration);
 }
 
+namespace {
+
+AcT toCellularAccessTechnology(int v) {
```

Comment:
What is `v`, "value"? Please change to a more verbose/descriptive variable name. Maybe something like, `rat_code`?

Reply:
I think `v` is ok here, `val` could have been better... but this function essentially tells you what v is... and technically it's just a number, but we do supply it here with input from a sscanf that happens to be an AcTStatus in one case and AcT in another case based on definition from u-blox modem.

Reply:
can we call it `act`? It's infinitely more descriptive than `v`.

Reply:
ended up with `rat`...